### DISCUSSION TITLE
Layout

### DISCUSSION THREAD

**Discussion by Stelvin004** [skyline-emu/skyline#2163] [Ideas]
Give an option to adjust the layout of on screen controller,most of the roms has freezing after 1625 update,when i reinstall 1625 version of skyline all the roms working properly like before please fix this issue it's snapdragon 778g processor,give an resolution options like Citra 1x,1.5x,2x like this

**Comment by TheASVigilante** [comment]
You can adjust the layout of the on-screen controls in the input settings for the 1# controller, and if you want to play in a higher resolution then activate docked mode.
